### DISCUSSION TITLE
Add fields from the Paleo Module to public search form

### DISCUSSION THREAD

**Discussion by ljwalker** [Symbiota/community-board#204] [Wish List]
Requested by a Pteridoportal user:

Fields developed for the Paleo Module are not included on the public search form.

Adding all or some of these fields to the public search form would greatly enhance the discoverability of fossil data in Symbiota portals.

Paleo Module: 
<img width="802" alt="Screen Shot 2022-09-02 at 11 04 18 AM" src="https://user-images.githubusercontent.com/11181096/188212838-d590b6e7-a89e-4a87-a8fc-fda99eeff43b.png">

**If adding paleo fields to the public search form is of interest to you, which fields would you like to see added?**

**Comment by ljwalker** [comment]
@joycegross - Was this explored when the paleo module was developed for PCC?

**Comment by JoyceGross** [comment]
As far as I remember, it was not discussed, at least not as part of the development. We were focused on being able to get the fossil data into the database, then there was/is the issue of the (missing) fossil names. It would be useful to be able to search on all the fossil-related fields. A sensible next step!

**Comment by catastropheelinggood** [comment]
Many of the fossils contributed by DMNS have only broad determinations and would be more accessible if searching by fossil-related locality data was possible. If I was interested in comparing fossil pteridos from the same formation or time period but from all institutions, I'm not seeing how I could do that currently.
